src/api/protectedApiClient.ts

```diff
@@ -117,6 +119,15 @@ export interface ProtectedApiClient {
     request: NameSiteEntryRequest,
   ) => Promise<void>;
   readonly addSites: (request: AddSitesRequest) => Promise<void>;
+  readonly sendEmail: (request: SendEmailRequest) => Promise<void>;
+  readonly filterSites: (
+    treeCommonNames: string[] | null,
+    adoptedStart: Date | null,
+    adoptedEnd: Date | null,
+    lastActivityStart: Date | null,
+    lastActivityEnd: Date | null,
+    neighborhoodIds: number[] | null,
+  ) => Promise<FilterSitesResponse>;
```

Comment:
Can we pull these fields out into its own type so it's easier to change the interface declaration, api route, and function implementation (maybe call it `FilterSitesRequest` or `FilterSitesParams`)? Also, could we maybe change the `Date` fields to just `string`, since that's how they're stored in the emailer filters state?

Reply:
Overlooked that I can just squish all of the args into a single object, good call!